### ISSUE TITLE
Verisign Source Plugin

### ISSUE THREAD

**Issue by kevgliss** [Netflix/lemur#69]
The verisign plugin should be able to poll the verisign API for all issued certificates and ensure that Lemur knows about them.

**Comment by JohnTheodore** [comment]
+1, this sounds awesome

**Comment by JohnTheodore** [comment]
@kevgliss Once you make new certs with lemur that seems fine, but did you do anything retroactively to import certs from the verisign api? a one off script to run, a hack or something?

**Comment by kevgliss** [comment]
@JohnTheodore I started this work quite a long time ago... See:
https://github.com/Netflix/lemur/blob/master/lemur/plugins/lemur_verisign/plugin.py#L225

At the time I ran into an bug with the Verisign API that prevented me from continue to develop the plugin (500 error when running the report I needed to import the certificates). I submitted a bug report to Symantec and was told the issue was fixed, but by then all of our certificates were already being tracked by Lemur so I haven't had the time to go back and finish the VerisignSource plugin up. 

If you would like to run with it, that would be awesome and should be relatively simple. I don't believe I will be able to work on this in the near term. (I put it on the backlog for 0.6.x)
